examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -603,14 +603,6 @@ void wfx_rsi_task(void * arg)
                     hasNotifiedIPV4 = false;
                 }
 #endif /* CHIP_DEVICE_CONFIG_ENABLE_IPV4 */
-                /*
```

Comment:
@sharad-patil24 this should be removed and added to another PR. It doesn't have anything to do in this PR

Reply:
I see this is for the new file. Will leave it as is for now.